utils/handleMessage.js

```diff
@@ -2,36 +2,28 @@ const { callSendAPI } = require("./callSendAPI");
 const { randomJokes } = require("./randomJokes");
 const MessageJson = require("./../config/services.json");
 
-// function handleMessage(senderPsid, receivedMessage) {
+const { responseFromWit } = require("../wit/wit_handler");
+const { Wit, log } = require("node-wit");
+
+const wit = new Wit({
```

Comment:
create a seperate file for this, export form that file and use it here so that we only one instance of the win in the whole project